Preparing for new plans (#1051)
dnsimple/dnsimple-support: content/articles/dnsimple-plans.markdown

---
title: DNSimple plans
excerpt: Understand what plan fits you best
categories:
- DNSimple
---

# DNSimple Plans

## All plans include

- Register, transfer, and renew domains. [See TLD prices](https://dnsimple.com/tlds)

> - $2 per month per authoritative DNS zone with Anycast DNS and multi-layer DDoS defense.

Reviewer: ```suggestion
- Primary or secondary DNS zones with Anycast DNS and multi-layer DDoS defense, billed monthly per zone.
```

> - $2 per month for domain email forwarding.

Reviewer: ```suggestion
- Email forwarding, billed monthly per domain.
```

## Solo

No base subscription fee.

DNSimple's Solo plan provides anyone with a domain management platform where they can manage DNS, domains, and certificates with multiple providers through a single control plane.

## Teams

> $199 per month base subscription fee.

Reviewer: ```suggestion
Requires base monthly subscription fee.
```

Activates the following features:
- HTTPS URL redirects.
- Let's Encrypt custom hostname certificates.
- Wildcard certificates.
- Specify different roles for your team members on a per-domain basis.
- Regional and POOL records.
- Google Workspace user provisioning.
- Unlimited activity history.
- 99% SLA.

DNSimple's Team plan provides startups and small engineering teams with all the tools they need to buy, connect, and operate their domains and DNS across multiple providers.

## Enterprise

Custom subscription fee.

Activates the following features:
- Priority support
- Single sign-on with Okta or Google Workspace
- Vanity Name Servers.
- High-volume zones.

> - Discounted domain registration, transfer, and renewal fees.

Reviewer: ```suggestion
- Discounted domain registration, transfer, and renewal fees for high-volume TLDs.
```

> - DNS zone discounts.

Reviewer: ```suggestion
- DNS zone discounts available.
```

> - 100% SLA.

Reviewer: ```suggestion
- 100% SLA available.
```

---
title: DNSimple plans
excerpt: Understand what plan fits you best
categories:
- DNSimple
---

# DNSimple Plans

Please take a look at [the features listed for each plan on our pricing page](https://dnsimple.com/pricing#features).

## All plans include

- Register, transfer, and renew domains. [See TLD prices](https://dnsimple.com/tlds)
- Primary or secondary DNS zones with Anycast DNS and multi-layer DDoS defense, billed monthly per zone.
- Email forwarding, billed monthly per domain.

## Solo

No base subscription fee.

DNSimple's Solo plan provides anyone with a domain management platform where they can manage DNS, domains, and certificates with multiple providers through a single control plane.

## Teams

> Requires base monthly subscription fee.

Reviewer: ```suggestion
Requires a base monthly subscription fee.
```

Activates the following features:
- HTTPS URL redirects.
- Let's Encrypt custom hostname certificates.

> - Wildcard certificates.

Reviewer: ```suggestion
- Let's Encrypt wildcard certificates.
```
